boot/zephyr/Kconfig

```diff
@@ -13,6 +13,13 @@ config MCUBOOT
 	bool
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 
+# When compiling MCUBoot, the image will be linked to the boot partition.
+# Override .text offset to make sure it is set to zero.
+# This is necessary when other bootloaders set a different default for
+# application images which are not bootloaders.
```

Comment:
I wonder if it would be better to have

```
if BOARD_HAS_NRF5_BOOTLOADER
...
```

that way everything is left unaffected when not building for the PCA10059

Reply:
yes, will be safer

Reply:
Please recheck.